cmdeploy/src/cmdeploy/dns.py

```diff
@@ -44,11 +43,12 @@ def get(self, typ: str, domain: str) -> str | None:
 
     def check_ptr_record(self, ip: str, mail_domain) -> bool:
         """Check the PTR record for an IPv4 or IPv6 address."""
-        result = self.shell(f"dig -r -x {ip} +short").rstrip()
+        result = self.shell(f"dig @ns1.your-server.de -r -x {ip} +short").rstrip()
```

Comment:
This is only temporary, because some upstream DNS server cached the wrong PTR record and I don't want to wait 24 hours to proceed. -.-